operator/pkg/controlplane/etcd/mainfests.go

```diff
@@ -44,7 +44,8 @@ spec:
         - --key-file=/etc/karmada/pki/etcd/etcd-server.key
         - --data-dir=/var/lib/etcd
         - --snapshot-count=10000
-        - --log-level=debug
+        - --log-level=debug=
```

Comment:
Does removing = here have any effect? @zhzhuang-zju

Reply:
Thank you for finding this problem. I fixed it in PR #4286 . Can you help with review?